diff -Naur compiz-0.9.8.6.orig/plugins/composite/src/screen.cpp compiz-0.9.8.6/plugins/composite/src/screen.cpp
--- compiz-0.9.8.6.orig/plugins/composite/src/screen.cpp	2012-11-21 04:38:40.000000000 +0100
+++ compiz-0.9.8.6/plugins/composite/src/screen.cpp	2013-08-07 19:41:04.849840792 +0200
@@ -118,7 +118,7 @@
 	    if (event->xclient.message_type == Atoms::winOpacity)
 	    {
 		w = screen->findWindow (event->xclient.window);
-		if (w && (w->type () & CompWindowTypeDesktopMask) == 0)
+		if (w)
 		{
 		    unsigned short opacity = event->xclient.data.l[0] >> 16;
 
diff -Naur compiz-0.9.8.6.orig/plugins/composite/src/window.cpp compiz-0.9.8.6/plugins/composite/src/window.cpp
--- compiz-0.9.8.6.orig/plugins/composite/src/window.cpp	2012-11-21 04:38:40.000000000 +0100
+++ compiz-0.9.8.6/plugins/composite/src/window.cpp	2013-08-07 19:42:16.184936166 +0200
@@ -46,9 +46,8 @@
     }
 
     priv->opacity = OPAQUE;
-    if (!(w->type () & CompWindowTypeDesktopMask))
-	priv->opacity = s->getWindowProp32 (w->id (),
-					    Atoms::winOpacity, OPAQUE);
+    priv->opacity = s->getWindowProp32 (w->id (),
+	      			        Atoms::winOpacity, OPAQUE);
 
     priv->brightness = s->getWindowProp32 (w->id (),
 					   Atoms::winBrightness, BRIGHT);
@@ -482,9 +481,6 @@
 {
     unsigned short opacity;
 
-    if (priv->window->type () & CompWindowTypeDesktopMask)
-	return;
-
     opacity = screen->getWindowProp32 (priv->window->id (),
 					     Atoms::winOpacity, OPAQUE);
 
